android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -101,14 +117,65 @@ protected void onResume() {
         realmRepo.getSessionsByLocation(location)
                 .addChangeListener((sessions, orderedCollectionChangeSet) -> {
                     mSessions.clear();
+                    String upcomingTitle="";
+                    String track="";
+                    String color;
```

Comment:
The track is being used out of the loop, so it will go out of scope if I declare it in the loop.